atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -123,11 +124,13 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji task ID: %s", self.koji_task_id)
 
-        try:
+        self.koji_build_id = self.workflow.exit_results.get(KojiImportPlugin.key)
+        if self.koji_build_id is None:
```

Comment:
Let's keep these checks similar to avoid unexpected behavior. Either `koji_tag` performs `if build_id is None` check, or this plugin performs `if not self.koji_build_id`.

Checking for falsy value (`if not foo`) actually checks for things like empty string and None. If, for some reason, `koji_import` returned an empty string, `koji_tag` would attempt to get value from `koji_promote`. But `exit_sendmail` would be perfectly happy with an empty string value.